src/test/java/com/cloudogu/scm/review/comment/service/CommentIndexerTest.java

```diff
@@ -262,6 +263,45 @@ void shouldIndexRepository() {
     }
   }
 
+  @Nested
+  class ReIndexRepositoryTaskTaskTests {
```

Comment:
Double Task :)

Reply:
Is there no negative test case? For example, if the commands are not supported.

Reply:
What do you mean by "Commands are not supported" ? I am just calling existing methods that themselves should be unit-tested.

Reply:
This indexer should currently only work for Git since Hg and Svn doesn't support pull requests. This is checked in the code but not tested here.